src/lib/y2storage/autoinst_proposal.rb

```diff
@@ -19,42 +19,29 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "yast"
```

Comment:
Strictly speaking, the `require "yast"` is needed for the subsequent requires to work (yes, I know we usually skip it, but we probably shouldn't). I don't see the point in removing this particular line.